ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/proxy/etcd-encryption/deprovision/cucushift-installer-rehearse-gcp-ipi-proxy-etcd-encryption-deprovision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-proxy-etcd-encryption-deprovision
+  steps:
+  - chain: gather-network
+  - ref: gather-gcp-console
+  - chain: gather
+  - ref: gather-proxy
+  - ref: ipi-deprovision-deprovision
+  - ref: gcp-deprovision-stacks
+  - ref: gcp-deprovision-s3buckets
```

Comment:
I did not see these ref are existing.

Reply:
Yes, I got the wrong copy from aws chain, just fixed.